tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -68,30 +49,29 @@ const expectTargetDetails = async (target) => {
  */
 const expectContacts = async (contacts, target) => {
   contacts = contacts.sort((a, b) => a.name > b.name ? 1 : -1);
-  expect(await element.all(by.css(`.aggregate-detail li`)).count()).toEqual(contacts.length);
+  expect(await analytics.getAggregateDetailListLength()).to.equal(contacts.length);
   // eslint-disable-next-line guard-for-in
   for (const idx in contacts) {
     const contact = contacts[idx];
-    const lineItem = element.all(by.css(`.aggregate-detail li`)).get(idx);
-    expect(await lineItem.getAttribute('data-record-id')).toEqual(contact._id);
-    expect(await lineItem.element(by.css('h4')).getText()).toEqual(contact.name);
-    expect(await lineItem.element(by.css('.detail')).getText()).toEqual(contact.counter);
-
+    const lineItem = await analytics.getAggregateDetailListElementbyIndex(idx);
+    const lineItemInfo = await analytics.getAggregateDetailElementInfo(lineItem);
```

Comment:
can this return a full analysis of the target item? Including the progress bar and goal, so to reduce the if/else below?